src/api/constants/application/index.ts

```diff
@@ -20,6 +20,13 @@ export const APPLICATION = {
   DEAL_TYPE: 'EXIP',
   SUBMISSION_COUNT_DEFAULT: 0,
   SUBMISSION_DEADLINE_IN_MONTHS: 1,
+  ALL_SECTIONS: 'all-sections',
+  SUBMISSION_DEADLINE_EMAIL: {
+    REMINDER_DAYS: 2,
+    EARLIER_LIMIT_TIME_SET: 0,
+    LATER_LIMIT_HOURS: 23,
+    LATER_LIMIT_TIME_SET: 59,
```

Comment:
`MINUTES` instead of `TIME_SET`? Also for `EARLIER`

Is there a reason for `EARLIER_LIMIT`/`LATER_LIMIT` naming? It's a start/end timeframe rather than a limit?